report/report.md

```
@@ -0,0 +1,602 @@
+    Tables --> Triggers
+    Tables --> Procedures
+    Tables --> Views
+    
+    style UI fill:#e3f2fd
+    style Routes fill:#fff3e0
+    style Tables fill:#e8f5e9
+```
+
+### 2.2 Technology Stack
+
+**Frontend (Next.js 14)**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)